.gitignore

```
@@ -17,3 +17,6 @@ ci-scripts/
 *.tar.gz
 
 /provider/cmd/pulumi-resource-*/schema.go
+# This is an artifact only so that we can pass the
+# schema for the provider GetSchema gRPC request
+/provider/cmd/pulumi-gen-*/schema.json
```

I may very well be missing something, but shouldn't this line be the following?

```suggestion
/provider/cmd/pulumi-resource-*/schema.json
```

---

good catch!